plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -427,6 +428,7 @@ protected void closeWriter(int writerIndex)
 
         writtenBytes += writer.getWrittenBytes() - currentWritten;
         memoryUsage -= currentMemory;
+        validationCpuNanos += writer.getValidationCpuNanos();
```

Comment:
This change does not make any difference because parquet writer validation works only in the Hive connector.